src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = sourceString.split(';');
```

Comment:
[CODE STYLE] - Avoid variable namings like `result` as it is too generic and does not describe the contents of the variable. Consider a more descriptive name that reflects that this is an array of style declarations, such as `styleDeclarations`.